test/e2e/nodenetworkstate_test.go

```diff
@@ -96,7 +96,6 @@ var _ = Describe("NodeNetworkState", func() {
 
 				// Let's clean the bridge directly in the node
 				// bypassing nmstate
-				deleteConnectionAtNodes("eth1")
```

Comment:
how is this related?

Reply:
Now eth1 is a real secondary nic from kubevirtci, to emulate a real outbound interface.

Reply:
We still create a connection profile for it, therefore we should clean up after the test is done. I will not remove the interface.

Reply:
We cannot remove it

Reply:
Why not? What happens if you do? Who handles clean-up after the test?

Reply:
Humm we should remove them, don't know why I delete the delete, adding them back.